test/suites/standard2/workload-partitioning.robot

```diff
@@ -45,3 +230,29 @@ Oc Get All Pods
     ...    ${EMPTY}
     ...    range .items[*]}{\.metadata\.namespace}{"@"}{\.metadata\.name}{"\\n"}{end
     RETURN    ${data}
+
+Crio Process ID
+    [Documentation]    Return the current crio process ID
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    pidof crio
+    ...    return_stdout=True    return_stderr=True    return_rc=True
+    Log    ${stderr}
+    RETURN    ${stdout}
+
+Remove Files
+    [Documentation]    removes files from the microshit host
+    [Arguments]    @{files}
+    Log    ${files}
+    ${files_path}=    Catenate    SEPARATOR=${SPACE}    @{files}
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    rm -f ${files_path}
+    ...    sudo=True    return_stdout=True    return_stderr=True    return_rc=True
+    Should Be Equal As Integers    ${rc}    0
+
+Cleanup And Create NS
+    [Documentation]    cleanup microshift and recreate the namespace for workloads
+    Cleanup MicroShift    --all    --keep-images
```

Comment:
Keep in mind this line will eliminate /var/lib/microshift, meaning the kubeconfig in use is also gone and new certs are generated.

Reply:
do we  have some other way to remove all the running containers ?

Reply:
This is ok :) what I mean is that you need to setup kubeconfig again after doing a cleanup to have things under control.

Reply:
yes, thats handled in `Restart MicroShift`  [keyword](https://github.com/openshift/microshift/blob/main/test/resources/microshift-process.resource#L88)